Modules/IO/MeshBase/include/itkMeshConvertPixelTraits.h

```diff
@@ -170,15 +170,15 @@ ITK_DEFAULTCONVERTTRAITS_NATIVE_SPECIAL(double);
     {                                                                   \
       return pixel[0];                                                  \
     }                                                                   \
-  };
+  }
 
 
 // Define traits for Offset<> from dimensions 1 to 5
-ITK_MESH_DEFAULTCONVERTTRAITS_OFFSET_TYPE(1)
-ITK_MESH_DEFAULTCONVERTTRAITS_OFFSET_TYPE(2)
-ITK_MESH_DEFAULTCONVERTTRAITS_OFFSET_TYPE(3)
-ITK_MESH_DEFAULTCONVERTTRAITS_OFFSET_TYPE(4)
-ITK_MESH_DEFAULTCONVERTTRAITS_OFFSET_TYPE(5)
+ITK_MESH_DEFAULTCONVERTTRAITS_OFFSET_TYPE(1);
```

Comment:
Here, semicolons are added. Is this intentional?

Reply:
Yes. My understanding is that ITK prefers that even non-function-like macros should require trailing semis (by means of that static_assert hack).  Not a fan of that style personally, but going with it.